docker/kernel.sls

```diff
@@ -25,5 +25,8 @@ docker-dependencies-kernel:
     - require_in:
       - pkg: docker package
     - onlyif: dpkg --compare-versions {{ grains["kernelrelease"] }} lt 3.8
+    {%- if docker.proxy %}
+    - proxy: {{ docker.proxy }}
+    {%- endif %}
```

Comment:
I think this one is wrong? `pkg.installed` does not support `proxy`, am I right?

Reply:
My bad ... fixing it